.github/workflows/continuous-integration.yaml

```diff
@@ -1080,12 +1096,25 @@ jobs:
       # macOS builds are already long, so we decide not to use --weekly on them, it could be
       # changed. Remark also that, for mac, due to unexpected issues with GitHub, we have a
       # slightly different way to launch pytest
+      # Add support for re-running flaky tests on macOS (intel) CI 
+      # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/4428
       - name: PyTest Source Code
         run: |
           make pytest_macOS_for_GitHub
 
+
+  # Only send a report for the following CI:
+  # - when pushing to main
+  # - when pushing to a release branch
+  # - when running weekly tests
   send-report:
-    if: ${{ always() }}
+    if: |
```

Comment:
only report weekly CIs or push to main/release branch : to discuss but I feel it's a bit useless to report all regular CIs on slack